src/java.net.http/share/classes/jdk/internal/net/http/ResponseBodyHandlers.java

```diff
@@ -56,7 +52,7 @@ public final class ResponseBodyHandlers {
 
     private ResponseBodyHandlers() { }
 
-    private static final String pathForSecurityCheck(Path path) {
+    private static final String checkPathForDefaultFS(Path path) {
```

Comment:
Nit - we could probably inline the implementation of this method at the one single call site where this gets called. That way it's a bit more clear what this method is for.

Same comment for the same named method in one other class.

Reply:
Done.